cylc/uiserver/jupyterhub_config.py

```diff
@@ -49,21 +58,40 @@ def _load(path):
         exec(path.read_text())
 
 
-def load():
+def get_conf_dir_hierarchy(
+        config_paths: List[Path], filename: bool = True
+) -> List[Path]:
+    """Takes list of config paths, adds version and filename to the path"""
+    conf_hierarchy = []
+    version_hierarchy = get_version_hierarchy(hub_version or __version__)
```

Comment:
Use the package version here (no need for the switch):

```suggestion
    version_hierarchy = get_version_hierarchy(__version__)
```